plugin_wrapper/plugin_wrapper.go

```diff
@@ -208,12 +207,11 @@ func (pw *PluginWrapper) fetchPrices(symbols []string, ts int64) error {
 
 	report, err := pw.adapter.FetchPrices(symbols)
 	if err != nil {
-		pw.logger.Error("Fetch prices", "error", err.Error())
 		return err
 	}
 
-	if len(report.BadSymbols) != 0 {
-		pw.logger.Warn("find bad symbols: ", report.BadSymbols)
+	if len(report.UnRecognizeSymbols) != 0 {
```

Comment:
i think the correct word would be unrecognizable

Reply:
> i think the correct word would be unrecognizable

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63